### PR TITLE
(feat): Added a tab to show css file

### LINKED ISSUE #300: Show css or other output file when it is generated
The repl component can only show one code snippet, it would be great to have multiple tabs when for example `css-modules` are also generated.

### PR DESCRIPTION
**Pull request by JayaKrishnaNamburu**
Fixes #300

### CODE REVIEW

File: packages/teleport-repl-component/src/components/editor/editor.tsx [hunk 1]
```diff
@@ -16,13 +16,34 @@ export class Editor {
...
   }
 
+  public handleTabClick() {
+    const el = document.getElementById('css-tab')
+    const editor = document.getElementById('cssEditor')
+    this.showCssFile = !this.showCssFile
+    if (this.showCssFile) {
+      this.openTab()
+    } else {
+      editor.style.display = 'none'
```
Comment (alexnm) [inline packages/teleport-repl-component/src/components/editor/editor.tsx]: shouldn't these styles be set based on `showCssFile` inside the `render` function?

File: packages/teleport-repl-component/src/components/editor/editor.tsx [hunk 2]
```diff
@@ -16,13 +16,34 @@ export class Editor {
...
+      el.style.height = '20px'
+    }
+  }
+
+  public openTab() {
+    const el = document.getElementById('css-tab')
+    const editor = document.getElementById('cssEditor')
+    editor.style.display = 'block'
```
Comment (alexnm) [inline packages/teleport-repl-component/src/components/editor/editor.tsx]: same question as above

File: packages/teleport-repl-component/src/components/editor/editor.tsx (hunk 1 above)
Comment (JayaKrishnaNamburu) [reply packages/teleport-repl-component/src/components/editor/editor.tsx]: No, to get the render toggle functionality. On click of the header, I am listening to this func and setting the styles. I think stencil doesn't support for inline styles with state currently, so need to do using plain JS.

Comment (alexnm) [reply packages/teleport-repl-component/src/components/editor/editor.tsx]: the render function should be triggered any time you are changing something that is annotated with @State. so styles could be set based on that as well I think. have you tried it?

Comment (JayaKrishnaNamburu) [reply packages/teleport-repl-component/src/components/editor/editor.tsx]: Yes, inline styling like with state or prop is currently not supported, they support only class as of now

Comment (alexnm) [reply packages/teleport-repl-component/src/components/editor/editor.tsx]: then we could do class toggle, if state is X, add that class to the element

Comment (JayaKrishnaNamburu) [reply packages/teleport-repl-component/src/components/editor/editor.tsx]: Yeah makes sense, i will refactor by setting classes

**Pull request merged**